site_config/community.jsx

```diff
@@ -4,15 +4,6 @@ export default {
   'en-us': {
     barText: 'Community',
     sidemenu: [
-      {
-        title: 'Community News',
-        children: [
-          {
-            title: 'Community News',
-            link: '/en-us/community/index.html',
-          },
-        ],
-      },
```

Comment:
why do we need to delete this?

Reply:
Because the content of this module is duplicated with the content of the homepage, it has been deleted.